fabfile/utils.py

```diff
@@ -22,6 +22,10 @@ def psql(command):
     with cd(config.paths['pg-latest']):
         return run('bin/psql -c "{}"'.format(command))
 
+def psql_file(path):
+    with cd(config.paths['pg-latest']):
+        return run('bin/psql -f ' + path + ' | grep Time:')
```

Comment:
Could we get time in a more intuitive way? I.e., could we get time from `run` command?